frontend/.npmrc

```diff
@@ -0,0 +1,2 @@
+@fortawesome:registry=https://npm.fontawesome.com/
+//npm.fontawesome.com/:_authToken=7FADC5BA-2678-4A03-95C9-A4B3F422FCFF
```

Comment:
I see in https://github.com/iFixit/ifixit/commit/e7cc5a4262380c8c350385b868859269cb3b6409 we also have this line:

```
//npm.fontawesome.com/:always-auth=true
```

Do we want that here too? Not really sure what this authentication is for, but thought I'd point it out